SpotifyAPI/Web/Auth/WebApiFactory.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using SpotifyAPI.Web.Enums;
+using SpotifyAPI.Web.Models;
+
+namespace SpotifyAPI.Web.Auth
+{
+    public class WebApiFactory
```

Comment:
I'm mostly following the default ReSharper settings as well, but the `API` lasted from the beginning. :stuck_out_tongue_winking_eye: 

Gonna merge this tomorrow :+1: